sdk/node/src/api/mockHsmKeys.js

```diff
@@ -5,7 +5,7 @@ const shared = require('../shared')
   * Cryptographic private keys are the primary authorization mechanism on a
   * blockchain. For development environments, Chain Core provides a convenient
   * MockHSM.
-  * <br/><br/>
+  * 
```

Comment:
It looks to me like almost 25% of this PR is unrelated edits like this one. It would be nice to keep each PR focused on one thing when possible.